chips/lowrisc/src/registers/adc_ctrl_regs.rs

```diff
@@ -0,0 +1,133 @@
+// Generated register constants for adc_ctrl.
+// This file is licensed under either of:
+//   Apache License, Version 2.0 (LICENSE-APACHE <http://www.apache.org/licenses/LICENSE-2.0>)
+//   MIT License (LICENSE-MIT <http://opensource.org/licenses/MIT>)
+
+// Built for earlgrey_silver_release_v5-5654-g222658011
+// https://github.com/lowRISC/opentitan/tree/222658011c27d6c1f22f02c7f589043f207ff574
```

Comment:
I started looking at bumping the OT version that Tock supports

This SHA doesn't seem to be part of the master branch of OT, so we would have to wait for this commit to be merged first though